### DISCUSSION TITLE
Is there a way to list proxied packages on the UI?

### DISCUSSION THREAD

**Discussion by d70rr3s** [verdaccio/verdaccio#4542] [Q&A] [answered]
I started using Verdaccio recently and for some reason is proxying packages wonderfully but on the UI shows none. Does only display/search packages published through it? Would be great to have all packages listed on the UI.

**Comment by juanpicado** [comment, marked as answer]
There's a way, to be clear is not exactly what you request (this has been asked few times already, please search for more context).

In a nutshell:

- On version v5.x **only support private display and search (only private local packages, no uplinks/proxy)**
- On (future) [version 6.x](https://github.com/orgs/verdaccio/discussions/4018) is oriented to only support private display and **UI search either private or proxied/uplinks**. 

In any version  _if you have a private package and display it's dependencies by clicking on each of them you can easily navigate through them_, so in a way are accessible but not from the landing page.

Currently it is not on the roadmap list packages on the UI directly due performance concerns, maybe that change over time.
